hack/kind-up.sh

```diff
@@ -220,6 +220,28 @@ setup_kind_with_lpp_resize_support() {
   kubectl delete --ignore-not-found=true storageclass local-path
 }
 
+check_shell_dependencies() {
```

Comment:
Can we also check the `coreutils` version?

Reply:
Any idea how? `coreutils` contains more than 100 tools.

Reply:
We could pick `date` as an representative